### PR TITLE
Add UI for HomeScreen and App Drawer

### PR DESCRIPTION
**Pull request by SriramPatibanda**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [ ] I have added necessary documentation (if appropriate)

### Description
- Work Completed: 
   - Add UI for HomeScreen including the app drawer.
- Work In Progress: 
   - Addition and navigation of required screens from Drawer.
   - Change quiz icon in grid tile.
- Work for which assistance is required:
   - Use GraphQL: APIs and replace all the fake data.
   - APIs required for this module: **User Details, Upcoming quizzes and Past Quizzes**.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### Build Status
[pr-build-test](https://github.com/dscnitrourkela/project-maple/actions/workflows/pull_request.yml)

### CODE REVIEW

File: aptiche/lib/services/net/authservice.dart [hunk 1]
```diff
@@ -16,7 +16,7 @@ class AuthService extends GetxController {
             return const LoginView();
           } else {
             if (snapshot.hasData) {
-              return const HomeScreen();
+              return HomeScreen();
```
Comment (abhibhaw) [inline aptiche/lib/services/net/authservice.dart:19]: Why @SriramPatibanda why

Comment (SriramPatibanda) [reply aptiche/lib/services/net/authservice.dart:19]: what?

File: aptiche/lib/utils/string.dart [hunk 2]
```diff
@@ -1,3 +1,8 @@
 //SVG Strings
+
 const String loginScreenSVG = 'assets/svgs/login_screen.svg';
 const String dataEntryScreen = 'assets/svgs/data_entry_screen.svg';
+const String homeScreenSVG = 'assets/svgs/TakeAQuiz.svg';
+
+//PNG Strings
+const String quizPNG = 'assets/images/quiz.png';
```
Comment (nayakastha) [inline aptiche/lib/utils/string.dart:8]: image background fixed? shouldn't we ask for a better svg to AICHE?

File: aptiche/lib/widgets/HomeScreen/drawer.dart [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+            child: DrawerHeader(
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                mainAxisSize: MainAxisSize.max,
+                children: <Widget>[
+                  GestureDetector(
+                    onTap: () => Navigator.pop(context),
+                    child: const Icon(
+                      Icons.close,
+                      size: 40.0,
```
Comment (nayakastha) [inline aptiche/lib/widgets/HomeScreen/drawer.dart:37]: try adding dynamic sizes, if possible

File: aptiche/lib/widgets/HomeScreen/drawer_list_tile.dart [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(left: 10.0, bottom: 15.0),
+      child: ListTile(
+        title: Text(
+          text,
+          style: Theme.of(context).primaryTextTheme.bodyText2!.copyWith(
+                fontWeight: FontWeight.bold,
+                fontSize: 20,
```
Comment (nayakastha) [inline aptiche/lib/widgets/HomeScreen/drawer_list_tile.dart:18]: dynamic sizes here as well

**Review by nayakastha — CHANGES_REQUESTED** [review]

File: aptiche/lib/utils/string.dart (hunk 2 above)
Comment (SriramPatibanda) [reply aptiche/lib/utils/string.dart:8]: Yeah, they gave, I'll change it.

File: aptiche/lib/main.dart [hunk 5]
```diff
@@ -1,3 +1,4 @@
+import 'package:aptiche/services/net/authservice.dart';
```
[the remark this answers is not in the document]
Comment (SriramPatibanda) [reply aptiche/lib/main.dart:1]: Issue resolved.

File: aptiche/lib/widgets/HomeScreen/drawer_list_tile.dart (hunk 4 above)
Comment (SriramPatibanda) [reply aptiche/lib/widgets/HomeScreen/drawer_list_tile.dart:18]: Font sizes would be in this way only right. Size config here also? We didn't use them in theme defining na.

File: aptiche/lib/services/net/authservice.dart (hunk 1 above)
Comment (abhibhaw) [reply aptiche/lib/services/net/authservice.dart:19]: why u removed const, better setup named route na

**Comment by SriramPatibanda** [comment]
Requested changes implemented. Ready for re-review @abhibhaw @nayakastha .

**Review by nayakastha — APPROVED** [review]

**Review by nayakastha — APPROVED** [review]

**Pull request merged**